euphonic/qpoint_phonon_modes.py

```diff
@@ -339,9 +339,9 @@ def calculate_debye_waller(
 
         .. math::
 
-          \\textbf{W}_{\\kappa} =
+          W^{\\alpha\\beta}_{\\kappa} =
           \\frac{\\hbar}{4M_{\\kappa}\\sum\\limits_{\mathbf{q}}{weight_\mathbf{q}}}
-          \\sum\\limits_{\mathbf{q}\\nu \in{BZ}}weight_\mathbf{q}\\frac{\\mathbf{e}_{\mathbf{q}\\nu\\kappa}\\mathbf{e}^{*}_{\mathbf{q}\\nu\\kappa}}
+          \\sum\\limits_{\mathbf{q}\\nu \in{BZ}}weight_\mathbf{q}\\frac{e_{\mathbf{q}\\nu\\kappa\\alpha}e^{*}_{\mathbf{q}\\nu\\kappa\\beta}}
           {\\omega_{\mathbf{q}\\nu}}
           coth(\\frac{\\hbar\\omega_{\mathbf{q}\\nu}}{2k_BT})
```

Comment:
Could the `coth` be `\mathrm{coth}`? Trig functions are usually typeset upright.

I'm not wild about _weight_ but it is difficult to suggest a non-confusing alternative.

Reply:
Done!